echion/signals.h

```diff
@@ -47,5 +47,6 @@ void install_signals()
 void restore_signals()
 {
     signal(SIGQUIT, SIG_DFL);
-    signal(SIGPROF, SIG_DFL);
+    if (native)
```

Comment:
Trying to remember why this would only be relevant for native 🤔

Reply:
Not sure either. @sanchda do you have any insights?

Reply:
IIRC, in pure-python mode we schedule our own sampling, so we don't need to set anything that emits a SIGPROF.

In full generality, we should probably sigaction to acquire the original signal disposition, then restore _that_.  :shrug:

Reply:
Ah right. The sigprof is used to get libunwind to run in threads to capture their stacks, so this makes sense